apps/ui/src/routes/common.ts

```diff
@@ -25,6 +25,7 @@ export const spaceChildrenRoutes: RouteRecordRaw[] = [
     component: SpaceEditor
   },
   { path: '', name: 'space-overview', component: SpaceOverview },
+  { path: 'about', redirect: { name: 'space-overview' } },
```

Comment:
Is this needed? This already direct me to space overview: https://snapshot.org/#/fabien.eth/about

Reply:
This was handled in the previous code (which is changed now to make it more readable)
```ts
redirectPath = `/${metadataNetwork}:${domain}`;
if (rest && !/^\/about$/.test(rest)) {
  redirectPath += rest;
}
```